Fly_Swatter/radar.py

```diff
@@ -17,13 +17,21 @@ def __init__(self, phi, theta, r, time_spot):
   def tell(self):
     return (f"Phi: {self.phi}  " + f"Theta: {self.theta} " + f"Rho: {self.r}")
 
+class velocity_vector():
```

Comment:
new velocity vector class enables control of direction by contacts; probably makes future simulation slightly more complicated and will probably be reworked to a flight path / dynamic system , along with a updating system for current derivative